hnn_core/viz.py

```diff
@@ -1189,6 +1189,9 @@ def plot_laminar_csd(times, data, contact_labels, ax=None, colorbar=True,
         Sampling times (in ms).
     data : array-like, shape (n_channels, n_times)
         CSD data, channels x time.
+    tmin : float | None
+        Start time of plot in milliseconds. If None, plot entire simulation.
+    tmax : float | None
```

Comment:
Not sure about this either. Is it necessary?

Reply:
Also deleted them too.